packages/cli/test/integration-2.test.ts

```diff
@@ -967,6 +805,12 @@ test('deploy pnpm twice using pnp and symlink=false', async () => {
   text = await page.text();
 
   expect(text).toContain('cache exists\n');
+
+  // Since this test asserts that we can create a new project based on the folder name, delete it after the test
+  // to avoid polluting the project list.
+  await apiFetch(`/projects/${session}`, {
+    method: 'DELETE',
+  });
```

Comment:
There's remaining work to be done to ensure that tests which create new projects on every run are also removing those projects as part of their cleanup, this issue is mostly ignored in the scope of this PR but here's an example of the kind of thing we'll need to do more of.